### PR TITLE
Updated query filter

### PR DESCRIPTION
**Pull request by mtruj013**
## Done

- Updated filter to match work done in [this PR](https://github.com/canonical-web-and-design/ubuntu.com/pull/11004)
- Made search queries case insensitive
- Added ability to search by title

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8030/security/api/docs
- Run a get request for a specific notice:
  - Search for runc and runC and ensure they both return the same thing
  - Attempt to search for notice by title and see that it is returned correctly

### CODE REVIEW

**Comment by carkod** [comment]
@mtruj013 do you have an issue to attach this to?

**Comment by mtruj013** [comment]
> @mtruj013 do you have an issue to attach this to?

No, I linked both of the original issues related to this on this [PR](https://github.com/canonical-web-and-design/ubuntu.com/pull/11004), this is a replica of what I pushed to the ubuntu.com repo so that they match. I'll make and link a new issue explaining as much, probs better that way

File: webapp/views.py [hunk 1]
```diff
@@ -424,8 +424,9 @@ def get_notices(**kwargs):
     if details:
         notices_query = notices_query.filter(
             or_(
-                Notice.id.like(f"%{details}%"),
-                Notice.details.like(f"%{details}%"),
+                Notice.id.ilike(f"%{details}%"),
+                Notice.details.ilike(f"%{details}%"),
+                Notice.title.ilike(f"%{details}%"),
                 Notice.cves.any(CVE.id.like(f"%{details}%")),
```
[the remark this answers is not in the document]
Comment (albertkol) [reply webapp/views.py:430]: My god.. I reviewed this days ago and I never submit the comment. :facepalm:

[the remark this answers is not in the document]
Comment (albertkol) [reply webapp/views.py:430]: Hmm.. from my QA with or without the `ilike` for `CVE.id` the API returns the same data. Hmm.. not sure why. I will make the change for the sake of consistency.

**Review by albertkol — APPROVED** [review]
:+1:

**Pull request merged**